public/src/app.scss

```diff
@@ -1,7 +1,7 @@
 @import 'bootstrap/scss/bootstrap.scss';
 @import '_variables_&_mixins.scss';
 
-@import 'Footer/_Footer.scss';
+@import 'shared/_footer.scss';
```

Comment:
just to make me happy can you do `./shared/_footer.scss`
think linux needs the `./`